### DISCUSSION TITLE
CLI wildcard fail

### DISCUSSION THREAD

**Discussion by protofolius** [comictagger/comictagger#504] [Q&A] [answered]
With the lack of masstagging GUI options , I am trying CLI but have an issue with it .
When tagging 1 file , it works . But from the moment I am using any wildcard , it fails .

1 file :
ComicTagger -s -t cr -m "publisher=Dupont" "421 #000 Title.cbr"
Save complete.

wildcard :
ComicTagger -s -t cr -m "publisher=Dupont" *.cbr
2023-09-01T09:17:02 | comictaggerlib.cli | ERROR | Cannot find *.cbr

instead of using "*.cbr" I have used many many variations like 
*.*
421*.cbr
"*"
`*.*`
"V:\Comics\421\*"

All of the commands fail with the "Cannot find x" error .
Which means I am still able to only tag 1 file at a time , same as in the GUI .

**Comment by lordwelch** [comment, marked as answer]
This is because you are on windows. Windows does not provide globbing, Linux, macOS and just about every other OS does it for the application.

You can use the `  -g, --glob            Windows only. Enable globbing` flag to enable globbing when on windows.

**Comment by protofolius** [comment]
I am using the command on windows indeed (tried both powershell and cmd) .
I did not expect a simple switch to fix it , but it did .

add -g to the command(s) and ... it works !

Thank you

**Comment by protofolius** [comment]
One more thing : The field "Story Arc" , can not be masstagged through cli ?

`comictagger.exe: error: argument -m/--metadata: 'story arc' is not a valid tag name`


EDIT : found out that it works by calling the field as story_arc and it the newer beta version it is story_arcs

**Reply by lordwelch** [reply]
Yes, most of the fields can be seen here https://github.com/comictagger/comictagger/wiki/UserGuide#new-template-format there are a few listed there that cannot be assigned to but should otherwise work. Note in the next alpha story_arc will be changed to story_arcs
